src/Sulu/Bundle/AdminBundle/Resources/js/components/Url/Url.js

```diff
@@ -176,6 +178,7 @@ export default class Url extends React.Component<Props> {
                     </SingleSelect>
                 </div>
                 <input
```

Comment:
That is because there is a single select right next to it. The standard `Input` component only accepts an Icon there, and we decided to implement an own component instead of bloating the existing one.

Reply:
actually i meant replacing the `input-element` that starts on line 178 with our `input-component`. i was thinking about this because we need to duplicate the styling of the `input-element` right now. (see https://github.com/sulu/sulu/pull/4247/files#diff-deaa8eaf7282fca4f86e7db8a68d3aedL26)